doc/api/buffer.markdown

```diff
@@ -600,6 +653,47 @@ Example:
 
     // 0.3333333333333333
 
+### buf.readPointerLE(offset, length[, noAssert])
+### buf.readPointerBE(offset, length[, noAssert])
+
+* `offset` Number
+* `length` Number
+* `noAssert` Boolean, Optional, Default: false
+* Return: Number
+
+Reads a pointer from the buffer by dereferencing at the specified
+offset with specified endian format.
+
+A new Buffer instance is returned at the memory address pointed to
+by `buf`, with its length set to `length`. If the NULL pointer is
+read, then JS `null` is returned, and `length` is disregarded.
+
+Set `noAssert` to true to skip validation of `offset`. This means that `offset`
+may be beyond the end of the buffer. Defaults to `false`.
+
+__WARNING__: The "Pointer" family of functions relate to direct
+memory access, which can easily get you into trouble (segmentation
+faults, etc.) if you don't know what you're doing. Generally these
+functions are only used in relation to the `dlopen` and `ffi` modules.
```

Comment:
I'm having a hard time being okay with including methods in `Buffer` that could have this sort of effect. Though I'll defer to the @nodejs/tsc as to whether this is alright to be included or not.